app/views/shared/head/_lite_mode_css.html.erb

```diff
@@ -14,6 +14,14 @@
   body img {
     display: none;
   }
+	
+	body#podcast .subscribe a.u-url {
+		display: block;
+	}
+	
+	body#podcast .subscribe a.u-url + br {
+		display: none;
+	}
```

Comment:
It looks like maybe your editor is set to use tabs instead of spaces. Would you mind find/replacing those tabs with two spaces so that these new CSS rules line up with the others, please?